test/conftest.py

```diff
@@ -36,6 +36,8 @@ def pytest_addoption(parser):
     parser.addoption('--framework-version', default='1.10.0')
     parser.addoption('--processor', default='cpu', choices=['gpu', 'cpu'])
     parser.addoption('--py-version', default='3', choices=['2', '3'])
+    parser.addoption('--ecr-image', default=None)
```

Comment:
seems like this redundant with `--docker-base-name` and `--tag` - do we really need all of these?

Reply:
I think I can use tag for the functional tests and add --account-id instead of --ecr-image. That seems better.